packages/@ember/-internals/runtime/lib/mixins/array.js

```diff
@@ -1100,6 +1100,28 @@ const ArrayMixin = Mixin.create(Enumerable, {
     argument for any item in the array. This method is often simpler/faster
     than using a callback.
 
+    ```javascript
+    class Jedi {
+      constructor(name, dark) {
+        this.name = name;
+        this.dark = dark;
+      }
+    }
+    const jediKnights = [
+      new Jedi('Luke Skywalker', false),
```

Comment:
Please use three core team member's names, to be consistent with other parts of the guides. Thank you!

Reply:
You can get them from https://emberjs.com/team/ 😁